daemon/src/monitor.rs

```diff
@@ -808,6 +809,16 @@ impl xtra::Actor for Actor {
                             Some(params) => params,
                         };
 
+                        // NOTE: this is a band-aid fix.
+                        // It is possible for an attestation to be available when the refund
+                        // timelock has expired, for example, when the daemon goes down and is
+                        // restarted. In this case we want to prioritise
+                        // broadcasting the cet over the refund transaction.
+                        // We wait at least 5 seconds after the monitor actor is initialised before
```

Comment:
How did we choose this number? I would have thought that we need to at least put 30 seconds here to avoid the race condition.

Reply:
Our sync interval is 5 seconds and task is added before reinitialising monitoring so we would get at least one 1 check to see if there is a pending attestation. I feel 30 secs might be too high as the user will experience a fairly long delay before seeing an updated state on restart. Maybe 10 seconds?

Reply:
On the other hand, if we pick a too short interval, we might not actually fix the problem!

Let's go with 30s. If we want to optimise this, we should fix it properly by not making it time-dependent.